src/api.ts

```diff
@@ -17,7 +25,39 @@ const getInventory = async (path: string) => {
   return getResponseOrError(response);
 };
 
-const getResponseOrError = (response) => {
+const getTasks = async (path: string) => {
```

Comment:
like `tasks_get` here or just `tasksGet`? could be good for code readability. Was also thinking of some type of namespace Inventory / Tasks. But that probably will not fit well with out file structure.

Reply:
just `tasksWhatever` and `inventoryWhatever`, I think it could be could especially in api and helpers where the functions from different part of apps mix together (other options would be to have special api and helpers for each part)